src/libs/OptionsListUtils.ts

```diff
@@ -1234,7 +1278,7 @@ function getValidOptions(
             const shouldShowInvoiceRoom =
                 includeInvoiceRooms &&
                 ReportUtils.isInvoiceRoom(reportOption.item) &&
-                ReportUtils.isPolicyAdmin(reportOption.policyID ?? '', policies) &&
+                ReportUtils.isPolicyAdmin(reportOption.policyID ?? String(CONST.DEFAULT_NUMBER_ID), policies) &&
```

Comment:
We should return early in the method if the value is undefined

```suggestion
                ReportUtils.isPolicyAdmin(reportOption.policyID), policies) &&
```